deploy/ui/form.json

```diff
@@ -1122,6 +1122,50 @@
                           "displayWhen": "external",
                           "visible": false,
                           "fields": [
+                             {
+                              "label": "Enable Extension image stream",
+                              "type": "checkbox",
+                              "required": false,
+                              "description": "When set to true will configure the KIE Server with Extension Image Stream.",
+                              "jsonPath": "$.spec.objects.servers[*].build",
+                              "originalJsonPath": "$.spec.objects.servers[*].build",
+                              "fields": [
+                                  {
+                                  "label": "Enable Extension image stream",
+                                  "type": "fieldGroup",
+                                  "jsonPath": "$.spec.objects.servers[*].build",
+                                  "originalJsonPath": "$.spec.objects.servers[*].build",
+                                  "displayWhen": "true",
+                                  "visible": false,
+                                  "fields": [
+                                    {
+                                      "label": "Extension image stream tag",
+                                      "type": "text",
+                                      "required": true,
+                                      "jsonPath": "$.spec.objects.servers[*].build.extensionImageStreamTag",
+                                      "description": "ImageStreamTag definition for the image containing the drivers and configuration. For example, custom-driver-image:7.7.0.",
+                                      "default": "rhpam-kieserver-library=org.openshift.quickstarts:rhpam-kieserver-library:1.5.0-SNAPSHOT"
```

Comment:
what does this default field means here?
it is the default value for hte kie container deployment id, it should not be here.

Reply:
Yes, I will make the corrections

Reply:
@spolti changes are done.